idlerpg.py

```diff
@@ -31,6 +31,11 @@
 
 bot = Bot(
     case_insensitive=True,
+    activity=activity=discord.Game(
```

Comment:
`activity=activity=` I think you only want one of these. :)

Reply:
I should probably wake up before going on GitHub 😅 
Good thing you noticed